### PR TITLE
Jetpack: 10.5-a.3 

### PR DESCRIPTION
**Pull request by samiff**
#### Changes proposed in this Pull Request:

* Atomic-only Jetpack release version 10.5-a.3

#### Jetpack product discussion

n/a

#### Does this pull request change what data or activity we track or use?

n/a

#### Testing instructions:

* Review changelogs and JP readme.txt

### CODE REVIEW

**Review by kraftbj — DISMISSED** [review]

File: projects/plugins/jetpack/CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,29 @@
 
 ### This is a list detailing changes for all Jetpack releases.
 
+## 10.5-a.3 - 2021-12-14
+### Enhancements
+- JITM: allow sidebar JITMs marked as dismissible to be dismissed.
```
Comment (jeherve) [inline projects/plugins/jetpack/CHANGELOG.md:7]: I don't think "JITM" is really helpful for most users. I would move this to "Other".

File: projects/plugins/jetpack/CHANGELOG.md [hunk 2]
```diff
@@ -2,6 +2,29 @@
...
+- SSO: add filters for the error text when a local user cannot be found for a given WP.com account and for when SSO is disallowed when on a staging site.
```
Comment (jeherve) [inline projects/plugins/jetpack/CHANGELOG.md:8]: I would say "Secure Sign On" instead of SSO here.

File: projects/plugins/jetpack/CHANGELOG.md [hunk 3]
```diff
@@ -2,6 +2,29 @@
...
+
+### Improved compatibility
+- Cleanup jetpack version option on plugin deactivation.
```
Comment (jeherve) [inline projects/plugins/jetpack/CHANGELOG.md:11]: I would rephrase this to something like

```suggestion
- General: ensure Jetpack options are properly deleted when the plugin is deactivated.
```

File: projects/plugins/jetpack/CHANGELOG.md [hunk 4]
```diff
@@ -2,6 +2,29 @@
...
+- Notifications: remove legacy code supporting IE versions older than 9.
+- Pay with PayPal Widget: hide widget from Legacy Widget block.
+- Subscription Block: add 'Success Message Text' to the block settings.
```
Comment (jeherve) [inline projects/plugins/jetpack/CHANGELOG.md:14]: I would move this to the enhancements.

File: projects/plugins/jetpack/CHANGELOG.md [hunk 5]
```diff
@@ -2,6 +2,29 @@
...
+
+### Bug fixes
+- Admin Menu: add support for WPML menu structure.
```
Comment (jeherve) [inline projects/plugins/jetpack/CHANGELOG.md:17]: I would move this to "Other", since only Atomic site owners (who don't get to see the changelogs) are concerned by this.

File: projects/plugins/jetpack/CHANGELOG.md [hunk 6]
```diff
@@ -2,6 +2,29 @@
...
+- I18n: ensure various strings in JavaScript UIs are marked for translation.
```
Comment (jeherve) [inline projects/plugins/jetpack/CHANGELOG.md:18]: I would replace I18n by something more user-friendly:

```suggestion
- Internationalization: ensure that all text, including text added via JavaScript, can be translated.
```

**Review by jeherve — CHANGES_REQUESTED** [review]

**Review by jeherve — DISMISSED** [review]

**Review by jeherve — APPROVED** [review]

**Comment by anomiex** [comment]
r236637-wpcom
